src/components/appNavigation/TabsView.js

```diff
@@ -126,7 +126,9 @@ const TabsView = React.memo((props: TabViewProps) => {
         </TouchableOpacity>
       )}
       <Appbar.Content />
-      <Appbar.Action icon="menu" onPress={toggleSidemenu.bind(null, store)} color="white" />
+      <TouchableOpacity onPress={toggleSidemenu.bind(null, store)}>
+        <Icon name="settings" size={20} color="white" style={{ marginRight: 10 }} />
```

Comment:
move styles  into `withStyles(getStylesFromProps)`

Reply:
@AlexeyKosinski it is already done in tother PR. So it will be ,erged to the master.